sql/session.go

```diff
@@ -19,12 +19,20 @@ const (
 	QueryKey key = iota
 )
 
+// Client holds session user information.
+type Client struct {
+	// User of the session.
+	User string
+	// Address of the client.
+	Address string
+}
+
 // Session holds the session data.
 type Session interface {
 	// Address of the server.
 	Address() string
```

Comment:
I thought we don't need it.

Reply:
We need client's address for process list and audit logs that I'm currently adding. What we don't need is server address `Session.Address`. I've left as it is already implemented and would break backwards compatibility.